lib/fog/parsers/terremark.rb

```diff
@@ -1,26 +1,26 @@
 module Fog
   module Parsers
     module Terremark
-      autoload :Base, 'fog/parsers/terremark/base'
-      autoload :GetCatalog, 'fog/parsers/terremark/get_catalog'
-      autoload :GetCatalogItem, 'fog/parsers/terremark/get_catalog_item'
-      autoload :GetInternetServices, 'fog/parsers/terremark/get_internet_services'
-      autoload :GetKeysList, 'fog/parsers/terremark/get_keys_list'
-      autoload :GetNetworkIps, 'fog/parsers/terremark/get_network_ips'
-      autoload :GetNodeServices, 'fog/parsers/terremark/get_node_services'
-      autoload :GetOrganization, 'fog/parsers/terremark/get_organization'
-      autoload :GetOrganizations, 'fog/parsers/terremark/get_organizations'
-      autoload :GetPublicIps, 'fog/parsers/terremark/get_public_ips'
-      autoload :GetTasksList, 'fog/parsers/terremark/get_tasks_list'
-      autoload :GetVappTemplate, 'fog/parsers/terremark/get_vapp_template'
-      autoload :GetVdc, 'fog/parsers/terremark/get_vdc'
-      autoload :InstantiateVappTemplate, 'fog/parsers/terremark/instantiate_vapp_template'
-      autoload :InternetService, 'fog/parsers/terremark/internet_service'
-      autoload :Network, 'fog/parsers/terremark/network'
-      autoload :NodeService, 'fog/parsers/terremark/node_service'
-      autoload :PublicIp, 'fog/parsers/terremark/public_ip'
-      autoload :Task, 'fog/parsers/terremark/task'
-      autoload :Vapp, 'fog/parsers/terremark/vapp'
+      autoload :Base, File.expand_path('../terremark/base', __FILE__)
+      autoload :GetCatalog, File.expand_path('../terremark/get_catalog', __FILE__)
+      autoload :GetCatalogItem, File.expand_path('../terremark/get_catalog_item', __FILE__)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.